pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/VenueProvidersManagerV2/AllocineProviderParamaters/AllocineProviderParameters.tsx

```diff
@@ -0,0 +1,118 @@
+import * as pcapi from 'repository/pcapi/pcapi'
+import React, { useCallback, useState } from 'react'
+import AllocineProviderFormDialog from '../../VenueProvidersManager/AllocineProviderFormDialog/AllocineProviderFormDialog'
+import { Button } from 'ui-kit'
+import { ButtonVariant } from 'ui-kit/Button/types'
+import { IAllocineProviderParametersValues } from './types'
+import { IVenueProviderApi } from '../../VenueProvidersManager/CinemaProviderItem/types'
+import { getRequestErrorStringFromErrors } from '../../VenueProvidersManager/utils/getRequestErrorStringFromErrors'
+import { showNotification } from 'store/reducers/notificationReducer'
+import style from './AllocineProviderParameters.module.scss'
+import { useDispatch } from 'react-redux'
+
+export interface IAllocineProviderParametersProps {
+  venueProvider: IVenueProviderApi
+  afterVenueProviderEdit: (editedVenueProvider: IVenueProviderApi) => void
+}
+
+const AllocineProviderParameters = ({
+  venueProvider,
+  afterVenueProviderEdit,
+}: IAllocineProviderParametersProps): JSX.Element => {
+  const [isOpenedFormDialog, setIsOpenedFormDialog] = useState(false)
+  const dispatch = useDispatch()
```

Comment:
A priori le dispatch ne servant qu'à afficher les notifications on peut utiliser useNotification 
```suggestion
  const notification = useNotification()
```